client/mainwindow.cpp

```diff
@@ -585,6 +591,12 @@ bool MainWindow::saveAccessTokenToFile(const AccountSettings& account,
 bool MainWindow::saveAccessTokenToKeyChain(const AccountSettings& account,
                                            const QByteArray& accessToken, bool writeToFile)
 {
+    if (!Quotient::SettingsGroup("UI").value("use_keychain", true).toBool())
```

Comment:
```suggestion
    if (!Quotient::Settings().value("UI/use_keychain", true).toBool())
```
A tiny bit more optimal. `SettingsGroup` is better when you intend to use different values from the same group, `Settings` is just fine for one-off reads/writes.